src/core/CMakeLists.txt

```diff
@@ -451,6 +451,10 @@ SET(QGIS_CORE_SRCS
   fieldformatter/qgsvaluemapfieldformatter.cpp
   fieldformatter/qgsvaluerelationfieldformatter.cpp
 
+  geocms/qgsgeocmsconnection.cpp
+  geocms/qgsgeonodeconnection.cpp
+  geocms/qgsgeonoderequest.cpp
```

Comment:
See comments below - we used a more generic term CMS in the case that others may want to add similar integrations. I'm ok to rename this for GeoNode - we can always refactor later if needed.